changelog.md

```diff
@@ -24,6 +24,8 @@
 - The `not` operator is now always a unary operator, this means that code like
   ``assert not isFalse(3)`` compiles.
 
+- `getProjectPath`, `gorgeEx` now return the same at RT or CT (before, they returned
```

Comment:
What is RT or CT? (I know...) Keep the jargon to a minimum. Not everybody is expert and many people even don't want to be experts.